vm/issue_vm.go

```diff
@@ -0,0 +1,63 @@
+package vm
+
+import (
+	"fmt"
+	"os"
+	"bufio"
+	"runtime"
+)
+
+func InitIssueReportVM (dir string, args []string) (*VM, error) {
+	v, err := New(dir, args)
+	v.mode = TestMode
+
+	return v, err
+}
+
+func PrintError(v *VM) {
+	eval := v.mainThread.stack.top().Target
+	err, ok := eval.(*Error)
+	if !ok {
+		fmt.Println("No error detected")
+	}
+	fmt.Printf("# Help! %s\n", err.Type)
```

Comment:
I don't think this `Help!` is necessary. User might not appreciate our humor when they have issues on Goby.